src/pages/UserDashboard.tsx

```diff
@@ -20,6 +21,14 @@ export default function UserDashboard() {
 
   const organizations = user.organizations || [];
 
+  const { isMobile, setOpenMobile } = useSidebar();
+
+  const handleMobileNavigation = () => {
+    if (isMobile) {
+      setOpenMobile(false);
+    }
+  };
```

Comment:
why only in user dashboard? 🤔 

let's never automatically open the sidebar in the first place? 🤔 i don't see any use case for that....

cc: @bodhish